contributors/design-proposals/api-machinery/auditing.md

```diff
@@ -141,7 +141,7 @@ const (
     AuditNone AuditLevel = "None"
     // AuditMetadata provides basic level of auditing, logging data at HTTP level
     AuditMetadata AuditLevel   = "Metadata"
-    // AuditRequestBody provides Header level of auditing, and additionally
+    // AuditRequestBody provides Metadata level of auditing, and additionally
```

Comment:
This edit doesn't look right. RequestBody is a lot of info

@tallclair

Reply:
I think this is right. Yes, audit events can be quite large. There's an issue to limit it: https://github.com/kubernetes/kubernetes/issues/60432

Reply:
I was wrong here, was reading line 144 as a comment on 143, but it's a comment on 145. REverting